tests/manager/manager-var-dump-001.phpt

```diff
@@ -45,10 +45,10 @@ object(MongoDB\Driver\Manager)#%d (%d) {
       ["is_passive"]=>
       bool(false)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
new ReadPreference($readPreference, array( array("dc" => "nyc"), array("disk" => "ssd")));

We set these things as arrays. I think this should all be arrays in the debug output too.